src/app/components/speakers/speakers.js

```diff
@@ -1,47 +1,43 @@
-
 import React from 'react';
 import { speakersData } from "assets/data/speakers-info.js";
 import './speakers.scss'
 
-const Speaker = (name, title, photo, index) => {
-    return (
-        <div className="speaker-img-wrapper">
-            <img 
-                src = {require('assets/images/speaker-photos/'+photo+'.jpeg')}
-                alt = {name + "'s profile picture"}
-            />
-            <div className="text">
-                <h4>{name}</h4>
-                <p>{title}</p>
-            </div>
-        </div>
-    );
-}
+export default function Speakers() {
```

Comment:
Some things here:
- let's use the newer `const functionName = (params) => {...}` form for functions/function components; e.g. `const Speakers = () => {...}`
- stylistically, in files where one thing gets exported out (like this speakers component) i like putting exports like `export default Speakers;` at the bottom of the file so there's a uniform place to see what's being exported out; but, in files with multiple exports, like if we're exporting a bunch of functions, putting the `export` before the declaration like this would be okay